tools/accuracy_checker/accuracy_checker/preprocessor/craft_preprocessing.py

```diff
@@ -0,0 +1,84 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from ..config import NumberField
+from ..preprocessor import Preprocessor
+from ..utils import get_size_from_config
+
+
+class CRAFTResize(Preprocessor):
+    __provider__ = 'craft_resize'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'size': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination size for image."
+            ),
+            'dst_width': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination width for image."
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination height for image."
+            ),
+            'magnify_ratio': NumberField(
+                value_type=float, optional=True, default=1,
+                description="Ratio to magnify image"
+            )
+        })
+
+        return parameters
+
+    def configure(self):
+        self.dst_height, self.dst_width = get_size_from_config(self.config)
+        self.magnify_ratio = self.get_value_from_config('magnify_ratio')
+
+    def process(self, image, annotation_meta=None):
+        data = image.data
+        height, width, channels = data.shape
+
+        max_dst_size = max(self.dst_height, self.dst_width)
+        max_image_size = max(height, width)
+        target_size = self.magnify_ratio * max_image_size
+        if target_size > max_dst_size:
+            target_size = max_dst_size
+        scale = target_size / max_image_size
+
+        target_w, target_h = int(width * scale), int(height * scale)
+
+        resized_data = cv2.resize(data, (target_w, target_h), interpolation=cv2.INTER_LINEAR)
+
+        target_h32, target_w32 = target_h, target_w
```

Comment:
I do not see need to have extra preprocessing. It can be divided on 2 parts (aspect ratio resize + strided padding)

Reply:
Removed